### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mcdonnelldean**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,10 +29,12 @@ npm install seneca-entity
 ```
 
 ## Using Seneca Entity
+The code below shows how to load this plugin as well as an in memory store.
 
 ```js
 require('seneca')()
   .use('entity')
+  .use('mem-store')
```
Comment (mihaidma) [inline README.md]: was the default mem-store removed from seneca?

Comment (mcdonnelldean) [inline README.md]: @mihaidma Yes. I wonder, should we load it by default in this module? All of the store stuff is now out of Seneca.

Comment (mcdonnelldean) [inline README.md]: @mihaidma @rjrodger Thinking about this more. The original functionality of seneca was mem store included. The work was to move entities out. So by rights we should probably add mem store back in so it all works as expected with a flag to turn it off if that's needed.

Looking at nodezoo, it is pretty useful to have it in by default as it allows isolation mode to happen by just not loading the redis plugin. 

I will push 1.2.0 today with it as the default option before we do a release of seneca.

**Comment by mcdonnelldean** [comment]
Closing as it won't make sense post adding mem store.

**Pull request closed without merging**